make/prepare

```diff
@@ -173,6 +182,10 @@ def delfile(src):
             itemsrc=os.path.join(src,item)
             delfile(itemsrc)
 
+if not os.path.exists(JOB_LOG_DIR):
+    os.makedirs(JOB_LOG_DIR)
+mark_file(JOB_LOG_DIR, mode=0o655)
```

Comment:
Should the mode be 755?  655 is seldomly seen.

Reply:
You are right...